presto-native-execution/presto_cpp/main/types/tests/PrestoToVeloxExprTest.cpp

```diff
@@ -0,0 +1,121 @@
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "presto_cpp/main/types/PrestoToVeloxExpr.h"
+#include <gtest/gtest.h>
+#include "presto_cpp/main/common/Configs.h"
+#include "velox/common/base/tests/GTestUtils.h"
+#include "velox/functions/remote/client/Remote.h"
+#include "velox/functions/remote/server/RemoteFunctionThriftService.h"
+
+using namespace facebook::presto;
+using namespace facebook::velox;
+
+class RemoteFunctionTest : public ::testing::Test {
+ protected:
+  static void SetUpTestCase() {
+    memory::MemoryManager::testingSetInstance({});
+  }
+
+  void SetUp() override {
+    memoryPool_ = memory::MemoryManager::getInstance()->addLeafPool();
+    converter_ =
+        std::make_unique<VeloxExprConverter>(memoryPool_.get(), &typeParser_);
+
+    functionHandle = std::make_shared<protocol::RestFunctionHandle>();
+    functionHandle->functionId = "remote.testSchema.testFunction;BIGINT;BIGINT";
+    functionHandle->version = "v1";
+    functionHandle->signature.name = "testFunction";
+    functionHandle->signature.returnType = "bigint";
+    functionHandle->signature.argumentTypes = {"bigint", "bigint"};
+    functionHandle->signature.typeVariableConstraints = {};
+    functionHandle->signature.longVariableConstraints = {};
+    functionHandle->signature.variableArity = false;
+
+    expectedMetadata.serdeFormat = functions::remote::PageFormat::PRESTO_PAGE;
+
+    testExpr.functionHandle = functionHandle;
+    testExpr.returnType = "bigint";
+    testExpr.displayName = "testFunction";
+    auto cexpr = std::make_shared<protocol::ConstantExpression>();
+    cexpr->type = "bigint";
+    cexpr->valueBlock.data = "CgAAAExPTkdfQVJSQVkBAAAAAAEAAAAAAAAA";
+    testExpr.arguments.push_back(cexpr);
+
+    auto cexpr2 = std::make_shared<protocol::ConstantExpression>();
+    cexpr2->type = "bigint";
+    cexpr2->valueBlock.data = "CgAAAExPTkdfQVJSQVkBAAAAAAEAAAAAAAAA";
+    testExpr.arguments.push_back(cexpr2);
+  }
+
+  std::unique_ptr<config::ConfigBase> restSystemConfig(
+      const std::unordered_map<std::string, std::string> configOverride = {})
+      const {
+    std::unordered_map<std::string, std::string> systemConfig{
+        {std::string(SystemConfig::kRemoteFunctionServerSerde),
+         std::string("presto_page")},
+        {std::string(SystemConfig::kRemoteFunctionServerRestURL),
+         std::string("http://localhost:8080")}};
+
+    for (const auto& [configName, configValue] : configOverride) {
+      systemConfig[configName] = configValue;
+    }
+    return std::make_unique<config::ConfigBase>(std::move(systemConfig), true);
+  }
+
+  std::shared_ptr<protocol::RestFunctionHandle> functionHandle;
+  protocol::CallExpression testExpr;
```

Comment:
Could you add the json for this expression in `types/tests/data`? That way we can also test more expressions.